automation_tools/__init__.py

```diff
@@ -229,8 +229,10 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
     if isinstance(run_katello_installer, str):
         run_katello_installer = (run_katello_installer.lower() == 'true')
 
-    forwarders = run('cat /etc/resolv.conf | grep nameserver | '
-                     'awk \'{print $2}\'', quiet=True).split('\n')
+    forwarders = run(
+        'cat /etc/resolv.conf | grep "^nameserver" | awk \'{print $2}\'',
```

Comment:
Basically the change was `s/grep nameserver/grep "^nameserver"`. Other changes was for better formatting.